### PR TITLE
Extend users core page

### LINKED ISSUE #1493: The user core concepts page needs form fields laid out
https://fusionauth.io/docs/v1/tech/core-concepts/users doesn't have form fields explaining every setting, like the other high level core concepts page does. ( https://fusionauth.io/docs/v1/tech/core-concepts/applications for example). 

Should build that out.

### PR DESCRIPTION
**Pull request by ColinFrick**
I added the Admin UI description and folded the `Segmenting Users` section into the relevant tab description.

Closes #1493

### CODE REVIEW

File: site/docs/v1/tech/core-concepts/users.adoc [hunk 1]
```diff
@@ -264,8 +359,287 @@ This means you can't use tenants to model stores.
...
+
+The user data tab allows you to view the user data for a user.
+
+User data is a key value store that allows you to store arbitrary data about a user. This data is not used by FusionAuth, it is simply stored and returned when the user is retrieved. FusionAuth does not provide a UI for managing user data. It is intended to be used by your application utilizing the API.
+
+image::core-concepts/user-data.png[User data screen in the administrative user interface.,width=1200]
+
+=== Source
+
+The source tab allows you to view the user in JSON format. This is useful for debugging and troubleshooting.
```
Comment (mooreds) [inline site/docs/v1/tech/core-concepts/users.adoc:540]:
```suggestion
The source tab allows you to view the user account as read-only JSON. This is equivalent to retrieving the user via link:/docs/v1/tech/apis/users[the User API]. This is useful for debugging and troubleshooting.
```

**Review by mooreds — CHANGES_REQUESTED** [review]
what about edit user drop down in upper right on the details screen? Can you please document that. 

please add to the 'table column' lists where appropriate, explanations of the magnifying glass and trash icons (and any other icons/actions)

**Comment by ColinFrick** [comment]
@mooreds This is ready to re-review.

This depends on the https://github.com/FusionAuth/fusionauth-style/pull/30 PR, because the orange button color is not yet available. Should I change it to gray for now?

**Comment by mooreds** [comment]
Closing, going to release under https://github.com/FusionAuth/fusionauth-site/pull/2413 with a few small changes. But in general, looks great and will be a huge help to folks navigating the FusionAuth admin UI.

**Pull request closed without merging**